kikit/fab/neodenyy1.py

```diff
@@ -0,0 +1,164 @@
+import click
+from pcbnewTransition import pcbnew
+import csv
+import os
+import sys
+import shutil
+import re
+from pathlib import Path
+from kikit.fab.common import *
+from kikit.common import *
+from kikit.export import gerberImpl
+mm_ius = 1000000.0
+footprintRe = [
+    (re.compile(r'Capacitor_SMD:C_(\d+)_.*'), 'C_{}'),
+    (re.compile(r'Diode_SMD:D_(\d+)_.*'), 'D_{}'),
+    (re.compile(r'Inductor_SMD:L_(\d+)_.*'), 'L_{}'),
+    (re.compile(r'Resistor_SMD:R_(\d+)_.*'), 'R_{}'),
+    (re.compile(r'Crystal:Crystal_SMD_(.*?)_.*'), 'CRYSTAL_{}')
+]
+
+def noFilter(footprint):
+    return True
```

Comment:
This function is defined twice

Reply:
Since it is a general-purpose function, I would suggest moving it into the `fab.common` module.

Reply:
Moved